### ISSUE TITLE
Conflicting peer dependency GraphQL 16

### ISSUE THREAD

**Issue by LZL0** [apollographql/apollo-client#8991]
```
npm ERR! Could not resolve dependency:
npm ERR! peer graphql@"^14.0.0 || ^15.0.0" from @apollo/client@3.4.16
npm ERR! node_modules/@apollo/client
npm ERR!   @apollo/client@"^3.4.16" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: graphql@15.7.2
npm ERR! node_modules/graphql
npm ERR!   peer graphql@"^14.0.0 || ^15.0.0" from @apollo/client@3.4.16
npm ERR!   node_modules/@apollo/client
npm ERR!     @apollo/client@"^3.4.16" from the root project
```

GraphQL version 16 conflicts with apollo-client's GraphQL version.

**Comment by LZL0** [comment]
Related to #8989

**Comment by brainkim** [comment]
This should be fixed in 3.5. We haven’t set it to latest because of some checks we’re running but please try it.
